public/dir.js

```diff
@@ -70,10 +72,16 @@
         const buttons = document.getElementsByClassName('delete-button');
         for (let button of buttons) {
           button.addEventListener('click', (e) => {
-            let deleteRequest = new XMLHttpRequest();
-            deleteRequest.onload = removeListItem(e.currentTarget);
-            deleteRequest.open('DELETE', `${window.location.pathname}/${e.currentTarget.parentElement.firstElementChild.innerText}`, true);
-            deleteRequest.send();
+            if (window.confirm(`Are you sure you want to delete ${e.currentTarget.parentElement.firstElementChild.innerText}`)) {
+              let deleteRequest = new XMLHttpRequest();
+              deleteRequest.onload = removeListItem(e.currentTarget);
+              deleteRequest.onerror = () => {
+                //TODO: Other ways to feedback error?
```

Comment:
if there's a problem deleting, we should notify the user so they know their file is still there. This is very rare so an actual alert() is fine.

Reply:
OK, I'll add an alert here. Just a note though: If there's a problem when the user tries to delete a file, the file name/reference will never leave the list of files on server. 

Not saying we shouldn't reinforce it with a prompt, just that the file still being on the list should be a good hint that it wasn't actually deleted from server.